sui_core/src/authority/authority_store.rs

```diff
@@ -506,6 +516,15 @@ impl<const ALL_OBJ_VER: bool> SuiDataStore<ALL_OBJ_VER> {
         Ok(())
     }
 
+    pub fn insert_cert(
```

Comment:
I guess that is also "unsafe", as in it does not update effects, remove + add new locks or update indexes.